tests/test_interactions.py

```diff
@@ -37,26 +37,16 @@ def test_close_contacts():
                         6, 2, 6, 5, 1, 8, 6, 5, 7, 4])
 
 
-@pytest.mark.skip
 def test_hbonds():
     """H-Bonds test"""
-    hbonds_count = [hbonds(rec, mol)[2].sum() for mol in mols]
-    assert_array_equal(hbonds_count,
-                       [6, 7, 5, 5, 6, 5, 6, 4, 6, 5, 4, 6, 6, 5, 8, 5, 6, 6,
-                        6, 7, 6, 6, 5, 6, 7, 5, 5, 7, 6, 6, 7, 6, 6, 6, 6, 6,
-                        6, 5, 5, 6, 4, 5, 5, 6, 6, 3, 5, 5, 4, 6, 4, 8, 6, 6,
-                        6, 4, 6, 6, 6, 6, 7, 6, 7, 6, 6, 7, 6, 6, 6, 5, 4, 5,
-                        5, 6, 6, 6, 6, 6, 6, 4, 7, 5, 6, 6, 5, 6, 6, 5, 6, 5,
-                        6, 5, 5, 7, 7, 6, 8, 6, 4, 5])
+    hbonds_count = np.array([hbonds(rec, mol)[2].sum() for mol in mols])
```

Comment:
I think I prefer to test the actual values (compare the vector of ints) instead of >0 check. Would it be possible to create a baseline for each toolkit if there are differences?